tests/lib/cmsis_dsp/transform/testcase.yaml

```diff
@@ -47,7 +47,7 @@ tests:
       - CONFIG_CMSIS_DSP_TEST_TRANSFORM_RF32=y
   libraries.cmsis_dsp.transform.cf64:
     min_flash: 1024
-    min_ram: 64
+    min_ram: 96
```

Comment:
@ABOSTM  Does this really need more than 64K of RAM in several ones or only this particular platform?
Could you try to build in some more ARM Boards with 64K RAM and see what is going on?

Reply:
@ioannisg , as explained in https://github.com/zephyrproject-rtos/zephyr/issues/27849
there is a malloc of 64K so when you add stack, variables ... 
So for sure, 64K is not enough.

Reply:
agreed, thanks